doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/autoscaling.md

```diff
@@ -0,0 +1,91 @@
+(kuberay-autoscaler-discussion)=
+# Autoscaling
+This page discusses autoscaling in the context of Ray on Kubernetes.
+For details on autoscaler configuration see the {ref}`configuration guide<kuberay-autoscaling-config>`.
+
+(autoscaler-pro-con)=
+## Should I enable autoscaling?
+Ray autoscaler support is optional.
+Here are some considerations to keep in mind when choosing whether to use autoscaling.
+
+### Autoscaling: Pros
+**Cope with unknown resource requirements** If you don't know how much compute your Ray
+workload will require, autoscaling can adjust your Ray cluster to the right size.
+
+**Save on costs** Idle compute is automatically scaled down, potentially leading to cost savings,
+especially when you are using expensive resources like GPUs.
+
+### Autoscaling: Cons
+**Less predictable when resource requirements are known.** If you already know exactly
+how much compute your workload requires, it could make sense to provision a static Ray cluster
+of the appropriate fixed size.
+
+**Longer end-to-end runtime.** Autoscaling entails provisioning compute for Ray workers
+while the Ray application is running. On the other hand, if you pre-provision a fixed
+number of Ray workers, all of the Ray workers can be started in parallel, potentially reducing your application's
+runtime.
+
+**Resource usage.** The autoscaler itself consumes CPU and memory resources.
```

Comment:
I'd consider getting rid of this -- it is really in the noise :)

Reply:
done